exercises/proverb/Cargo.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "proverb"
+version = "0.0.0"
```

Comment:
I had to call it version 0 since we have no https://github.com/exercism/problem-specifications/blob/master/exercises/proverb/canonical-data.json existing. That means if we call it 1.0.0 and then a 1.0.0 comes into existence that we do not comply with, we may falsely believe ourselves to be compliant, which defeats the purpose of tracking compliance